### ISSUE TITLE
A dependency has broken the test task

### ISSUE THREAD

**Issue by nicksrandall** [babel/babel-library-boilerplate#146]
I suspect that a recent update to Babel has done this but if you clone this repo and immediately run gulp test you will get this error: 

```bash
SyntaxError: /Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/gulp-mocha/node_modules/gulp-util/node_modules/beeper/index.js: 'return' outside of function (9:1)
   7 | 	process.argv.indexOf('--beep=false') !== -1) {
   8 | 	module.exports = function () {};
>  9 | 	return;
     |  ^
  10 | }
  11 |
  12 | function beep() {
    at Parser.pp.raise (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/src/location.js:73:13)
    at Parser.pp.parseReturnStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/src/statement.js:247:74)
    at Parser.pp.parseStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/src/statement.js:77:19)
    at Parser.parseStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/plugins/flow.js:631:22)
    at Parser.pp.parseBlock (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/src/statement.js:387:21)
    at Parser.pp.parseStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/src/statement.js:93:19)
    at Parser.parseStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/plugins/flow.js:631:22)
    at Parser.pp.parseIfStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/src/statement.js:241:26)
    at Parser.pp.parseStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/src/statement.js:75:19)
    at Parser.parseStatement (/Users/nicksrandall/Documents/DomoGithub/playground/babel-library-boilerplate/node_modules/babel/node_modules/babel-core/lib/acorn/plugins/flow.js:631:22)
```

**Comment by jmeas** [comment]
I won't have time to look into this for the next few days. If anyone has the time to investigate, it would be much appreciated!

**Comment by sebmck** [comment]
It was a regression in 5.2.0 that has since been fixed in Babel 5.2.1, see babel/babel#1399 for more info.

**Comment by jmeas** [comment]
Thanks @sebmck !
